src/util/ip.ts

```diff
@@ -273,9 +273,6 @@ export const ip = {
     });
     return ipl >>> 0;
   },
-  fromLong: function (ipl: number): string {
```

Comment:
Why remove this code?

Reply:
This function is not being used by `ipMatch()`. I think there's no reason to keep this.